### DISCUSSION TITLE
Alternatives for other OS'es / platforms (MacOS kiddie)

### DISCUSSION THREAD

**Discussion by viktorashi** [eliboa/NxNandManager#136] [Q&A] [answered]
heyy i was even trying to compile from source cuz maybe it works on the mac but the `windres` got me going crazy so idk, maybe some other alternatives? Or is ok if I use [icongnito itself](https://github.com/blawar/incognito) in some way (not sure how) ? Just use it directly somehow?

**Comment by viktorashi** [comment]
from icognito: 

> NOTE: Atmosphere blocks writes to PRODINFO, so you must use ReiNX, SX OS, or a custom KIP to install this. Once installed, you can switch back to Atmosphere.

how would i write this custom KIP cuz im more than willing, Or could you point me to the point this repo that uses that project. I'll look it up myself when less tired prolly

**Comment by impeeza** [comment, marked as answer]
If you need to incognito you can use Incognito_RCM payload on your console, but is not supported anymore.

is recommended to use Atmosphère's Exosphere prodinfo blanking!
https://github.com/Atmosphere-NX/Atmosphere/blob/master/docs/features/configurations.md

**Reply by viktorashi** [reply]
oh but if i have a modchipped one i cant really do that huh?
thanks a ton! I was already looking into exosphere and DNS-MITM. So Im too tired to realise the differenence betwenn icognito and exosphere but imma get it in the morning, thanks a ton yooo!

**Reply by viktorashi** [reply]
also, is it recommend to use exosphere on sysnand or nah? idk what I did but got banned💀
